### PR TITLE
Store user preferences in a cookie rather than sessionStorage

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

**Comment by danni** [comment]
Personally I'd use localstorage.

**Comment by DanielHeath** [comment]
Localstorage would also work. As discussed - I can't think of a way that it would be meaningfully different.

**Comment by DanielHeath** [comment]
I've switched to localstorage as recommended. 

Slightly concerned that it shares storage between tabs, but I think that's enough of an edge-case for now.

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "history": "1.13.1",
     "immutable": "3.7.5",
     "js-yaml": "3.4.2",
+    "koekje": "0.0.4",
```
Comment (danni) [inline package.json]: Is this still needed?

File: src/storage.js [hunk 2]
```diff
@@ -3,16 +3,22 @@
...
-
 class Storage {
 
     static getItem(key: string): ?(string|number|boolean) {
-        return sessionstorage.getItem(key);
+        if (typeof localStorage == "undefined") {
+            return null;
+        }
+
+        return localStorage.getItem(key);
```
Comment (danni) [inline src/storage.js:13]: Out of interest why is this not super upsetting flowtype?

Comment (DanielHeath) [inline src/storage.js:13]: It knows about `localStorage` being a thing (and the rest of the DOM stuff)

File: package.json (hunk 1 above)
Comment (DanielHeath) [inline package.json]: Good catch, no.

**Comment by danni** [comment]
++

**Pull request merged**